cub/test/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ option(METAL_BUILD_EXAMPLES OFF)
 option(METAL_BUILD_TESTS OFF)
 CPMAddPackage("gh:brunocodutra/metal@2.1.4")
 
+CPMAddPackage(NAME NVTX GITHUB_REPOSITORY NVIDIA/NVTX GIT_TAG release-v3 DOWNLOAD_ONLY True)
+add_subdirectory(${NVTX_SOURCE_DIR}/c ${NVTX_BINARY_DIR}/c)
```

Comment:
Is this the best way to obtain a dependency without a top-level `CMakeLists.txt`?